inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -311,6 +316,13 @@ void LayerTestsCommon::ConfigureNetwork() {
             out.second->setPrecision(outPrc);
         }
     }
+
+    std::map<std::string, ngraph::PartialShape> inputShapes;
+    auto inputsDataMap = cnnNetwork.getInputsInfo();
+    for (auto&& inputDataMap : inputsDataMap) {
+        inputShapes[inputDataMap.first] = std::vector<ngraph::Dimension>(inputDynamicShape);
```

Comment:
this behavior is incorrect for nodes with multiply parament inputs
because you set same shapes for all inputs
it should be fixed after adding possibility to specify multiple input shapes

also i guess we no need to call reshape for not dynamic case

Reply:
The codes will be checked as you commented.
And calling reshape is updated when targetStaticShape is not empty.